modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,76 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding a machine set to your cluster using the `arm64` bootimage  
+
+To add `arm64` worker nodes to your heterogeneous cluster, you must create an Azure machine set that uses the `arm64` bootimage. To create your own custom machine set on Azure, see "Creating a machine set on Azure". 
+
```

Comment:
shouldn't `see "Creating a machine set on Azure"` actually link to it?

Reply:
As a docs guideline, we can't put `xrefs` in modules becuase it can  cause issues with builds and other links. The link is in the _Additional recourses_ section at the bottom